### ISSUE TITLE
Fetch GPG key / Submit GPG key

### ISSUE THREAD

**Issue by rommelfs** [MISP/MISP#598]
It seems that there is confusion between the two functions.
People paste their PGP key and click on 'Fetch GPG key' - which obviously doesn't work if they haven't uploaded their key to the key servers.

I propose to clarify the situation:

Submit -> Upload GPG key 
Fetch GPG key -> Fetch from keyserver

perhaps add a small explanation

**Comment by bane0** [comment]
Hello !

In addition to this clarification, I think that we should put somewhere that gpg keys without expiry date are NOT accepted in MISP (can we change this behaviour in Crypt_GPG ?). This is not easy to spot, as there are no traces of this in the log files, the UI just shows an "Invalid GPG Key". I was having problems adding some specific keys and  the ones that were not accepted in MISP do not have expiry date.

**Comment by RichieB2B** [comment]
I just upgraded to MISP 2.4.7 and was able to create a new user with a PGP key without expiration data. I am using channel://pear.php.net/Crypt_GPG-1.4.0

**Comment by iglocska** [comment]
Indeed, haven't encountered that before. Can you save PGP keys with an expiration date? Is it just one key that doesn't save or all? Is the key sign only by any chance?
